server/src/test/java/org/jboss/pnc/buildagent/server/ConcurrentStdInOutTest.java

```diff
@@ -41,10 +44,11 @@ public static void tearDown() {
     }
 
     @Test
-    public void simpleProcessOutput() throws IOException, InterruptedException {
+    @Ignore
```

Comment:
Why are you ignoring it?

Reply:
Thinking on it - you're right, probably ignore is the wrong option - I can either just let it run or remove the test - WDYT?

Reply:
Yeah I'm guessing it's just about letting it run and seeing if there's any errors being thrown.